src/libs/sensors-logging.ts

```diff
@@ -392,24 +461,52 @@ PlayResY: ${videoHeight}
+Style: CenterMid,Arial,${this.telemetryDisplayOptions.value.fontSize},${convertRGBToBGRColor(this.telemetryDisplayOptions.value.fontColor)},&H000000FF,${convertRGBToBGRColor(this.telemetryDisplayOptions.value.fontOutlineColor)},&H00000000,${this.telemetryDisplayOptions.value.fontBold ? '1' : '0'},${this.telemetryDisplayOptions.value.fontItalic ? '1' : '0'},${this.telemetryDisplayOptions.value.fontUnderline ? '1' : '0'},${this.telemetryDisplayOptions.value.fontStrikeout ? '1' : '0'},100,100,0,0,1,2,2,5,10,10,10,1
+Style: RightMid,Arial,${this.telemetryDisplayOptions.value.fontSize},${convertRGBToBGRColor(this.telemetryDisplayOptions.value.fontColor)},&H000000FF,${convertRGBToBGRColor(this.telemetryDisplayOptions.value.fontOutlineColor)},&H00000000,${this.telemetryDisplayOptions.value.fontBold ? '1' : '0'},${this.telemetryDisplayOptions.value.fontItalic ? '1' : '0'},${this.telemetryDisplayOptions.value.fontUnderline ? '1' : '0'},${this.telemetryDisplayOptions.value.fontStrikeout ? '1' : '0'},100,100,0,0,1,2,2,6,10,10,10,1
+Style: LeftTop,Arial,${this.telemetryDisplayOptions.value.fontSize},${convertRGBToBGRColor(this.telemetryDisplayOptions.value.fontColor)},&H000000FF,${convertRGBToBGRColor(this.telemetryDisplayOptions.value.fontOutlineColor)},&H00000000,${this.telemetryDisplayOptions.value.fontBold ? '1' : '0'},${this.telemetryDisplayOptions.value.fontItalic ? '1' : '0'},${this.telemetryDisplayOptions.value.fontUnderline ? '1' : '0'},${this.telemetryDisplayOptions.value.fontStrikeout ? '1' : '0'},100,100,0,0,1,2,2,7,10,10,10,1
+Style: CenterTop,Arial,${this.telemetryDisplayOptions.value.fontSize},${convertRGBToBGRColor(this.telemetryDisplayOptions.value.fontColor)},&H000000FF,${convertRGBToBGRColor(this.telemetryDisplayOptions.value.fontOutlineColor)},&H00000000,${this.telemetryDisplayOptions.value.fontBold ? '1' : '0'},${this.telemetryDisplayOptions.value.fontItalic ? '1' : '0'},${this.telemetryDisplayOptions.value.fontUnderline ? '1' : '0'},${this.telemetryDisplayOptions.value.fontStrikeout ? '1' : '0'},100,100,0,0,1,2,2,8,10,10,10,1
+Style: RightTop,Arial,${this.telemetryDisplayOptions.value.fontSize},${convertRGBToBGRColor(this.telemetryDisplayOptions.value.fontColor)},&H000000FF,${convertRGBToBGRColor(this.telemetryDisplayOptions.value.fontOutlineColor)},&H00000000,${this.telemetryDisplayOptions.value.fontBold ? '1' : '0'},${this.telemetryDisplayOptions.value.fontItalic ? '1' : '0'},${this.telemetryDisplayOptions.value.fontUnderline ? '1' : '0'},${this.telemetryDisplayOptions.value.fontStrikeout ? '1' : '0'},100,100,0,0,1,2,2,9,10,10,10,1
```

Comment:
I would recommend to avoid the amount of duplication code with a simple loop
```suggestion
const { fontSize, fontColor, fontOutlineColor, fontBold, fontItalic, fontUnderline, fontStrikeout} = this.telemetryDisplayOptions.value;
const styles = [
  'LeftBottom', 'CenterBottom', 'RightBottom',
  'LeftMid', 'CenterMid', 'RightMid',
  'LeftTop', 'CenterTop', 'RightTop'
].map((name, index) => {
  `Style: ${name},Arial,${fontSize},${convertRGBToBGRColor(fontColor)},&H000000FF,${convertRGBToBGRColor()},&H00000000,${fontBold ? '1' : '0'},${fontItalic ? '1' : '0'},${fontUnderline ? '1' : '0'},${fontStrikeout ? '1' : '0'},100,100,0,0,1,2,2,${index + 1},10,10,10,1`
})

const assFile = `
  [Script Info]
  Title: Cockpit Subtitle Telemetry file
  ScriptType: v4.00+
  WrapStyle: 0
  ScaledBorderAndShadow: yes
  YCbCr Matrix: TV.601
  PlayResX: ${videoWidth}
  PlayResY: ${videoHeight}
  [V4+ Styles]
  Format: Name, Fontname, Fontsize, PrimaryColour, SecondaryColour, OutlineColour, BackColour, Bold, Italic, Underline, StrikeOut, ScaleX, ScaleY, Spacing, Angle, BorderStyle, Outline, Shadow, Alignment, MarginL, MarginR, MarginV, Encoding
  ${styles}
  [Events]
  Format: Layer, Start, End, Style, Name, MarginL, MarginR, MarginV, Effect, Text`;

```
```

Reply:
Your suggestion is elegant, and it would save duplicated code. But for me it didn't work as expected (implementation below). 
The telemetry elements were concentrated on the center bottom (2nd on the styles mapping.. weird..) and the result on the player is shown on the image below the code.

`  const { fontSize, fontColor, fontOutlineColor, fontBold, fontItalic, fontUnderline, fontStrikeout} = this.telemetryDisplayOptions.value
    const styles = [
      'LeftBottom', 'CenterBottom', 'RightBottom',
      'LeftMid', 'CenterMid', 'RightMid',
      'LeftTop', 'CenterTop', 'RightTop'
    ].map((name, index) => {
      `Style: ${name},Arial,${fontSize},${convertRGBToBGRColor(fontColor)},&H000000FF,${convertRGBToBGRColor(fontOutlineColor)},&H00000000,${fontBold ? '1' : '0'},${fontItalic ? '1' : '0'},${fontUnderline ? '1' : '0'},${fontStrikeout ? '1' : '0'},100,100,0,0,1,2,2,${index + 1},10,10,10,1`
    })

    /* eslint-disable vue/max-len, prettier/prettier, max-len */
    let assFile = `
    [Script Info]
    Title: Cockpit Subtitle Telemetry file
    ScriptType: v4.00+
    WrapStyle: 0
    ScaledBorderAndShadow: yes
    YCbCr Matrix: TV.601
    PlayResX: ${videoWidth}
    PlayResY: ${videoHeight}
    [V4+ Styles]
    Format: Name, Fontname, Fontsize, PrimaryColour, SecondaryColour, OutlineColour, BackColour, Bold, Italic, Underline, StrikeOut, ScaleX, ScaleY, Spacing, Angle, BorderStyle, Outline, Shadow, Alignment, MarginL, MarginR, MarginV, Encoding
    ${styles.join('\n')}
    [Events]
    Format: Layer, Start, End, Style, Name, MarginL, MarginR, MarginV, Effect, Text`
`

![image](https://github.com/bluerobotics/cockpit/assets/14910201/806844b8-048a-4d16-b770-c6f22184fb32)

Reply:
It wasn't the typo, but the what the function was doing after mapping the styles. Corrected now